pkgs/development/python-modules/python-fluent/fluent-pygments.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage, fetchPypi, lib, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "fluent.pygments";
```

`pname`s should be normalized according PEP-503, see https://nixos.org/manual/nixpkgs/stable/#contributing

```suggestion
  pname = "fluent-pygments";
```